src/Common/src/System/Net/Internals/SocketException.Windows.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Net.Sockets;
+
+namespace System.Net.Internals
+{
+    internal class InternalSocketException : SocketException
+    {
+        // TODO: Add a public ctor to SocketException instead.
```

Comment:
Just curious, since I realize this is temporary, but this file looks identical to the .Unix.cs one... is there a reason they're kept distinct?